### PR TITLE
Populate InfoURL for nearstars.stc

### PR DESCRIPTION
**Pull request by levinli303**
Populated using a script:

1. collect all names
2. query the name in wikipedia to see if a page is available
3. get the canonical link to use in nearstars.stc

### CODE REVIEW

File: data/nearstars.stc [hunk 1]
```diff
@@ -2602,6 +2712,7 @@ Barycenter "Gliese 644 B:Wolf 630 B"
 	SpectralType "M4.0V"
 	Radius 180000 # approx. for class
 	AppMag 12.74
+	InfoURL "https://en.wikipedia.org/wiki/List_of_stars_in_Centaurus"
```
Comment (Askaniy) [inline data/nearstars.stc:2715]: https://en.wikipedia.org/wiki/GJ_3737

File: data/nearstars.stc [hunk 2]
```diff
@@ -2666,6 +2778,7 @@ Barycenter 72659 "XI Boo:37 Boo:Gliese 566:ADS 9413"
 	SpectralType "M4.5V"
 	Radius 160000 # approx. for class
 	AppMag 12.834
+	InfoURL "https://en.wikipedia.org/wiki/List_of_stars_in_Cancer"
```
Comment (Askaniy) [inline data/nearstars.stc:2781]: https://en.wikipedia.org/wiki/Gliese_299

File: data/nearstars.stc [hunk 3]
```diff
@@ -2996,6 +3112,7 @@ Modify 103096 # Gliese 809
 	SpectralType "M7.0V"
 	Radius 84000 # approx. for class
 	AppMag 17.141
+	InfoURL "https://en.wikipedia.org/wiki/List_of_stars_in_Hydra"
```
Comment (Askaniy) [inline data/nearstars.stc:3115]: https://en.wikipedia.org/wiki/LHS_3003

**Review by Askaniy — CHANGES_REQUESTED** [review]
Perhaps for stars without articles, use existing redirect links? Then we won't have to worry about updating.

**Review by Askaniy — APPROVED** [review]

**Comment by levinli303** [comment]
> Perhaps for stars without articles, use existing redirect links? Then we won't have to worry about updating.

For the ones that I have not filled in, there are no redirect links. Wikipedia gives a "Wikipedia does not have an article with this exact name".

**Pull request merged**